data/distro.yml

```diff
@@ -353,3 +353,36 @@
       </code></pre>
       <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
     </ol>
+
+- name: "Chrome OS"
+  logo: "chrome-os.svg"
+  info: >
+    <p>Flatpak applications can be installed on Chrome OS with the Crostini Linux compatibility layer. This is not available for all Chrome OS devices, so you should ensure your device is compatible before proceeding. A list of compatible devices is maintained <a href="https://www.reddit.com/r/Crostini/wiki/getstarted/crostini-enabled-devices">here</a>. Also note that prior to Version 69, Chrome OS required to be changed to the <strong>Developer channel</strong> before Crostini could be installed.</p>
+    <ol class="distrotut">
+      <li>
+        <h2>Enable Linux support</h2>
+        <p>Navigate to <a href="chrome://settings">chrome://settings</a>, and scroll down to <strong>Linux (Beta)</strong> and turn it on. Chrome OS will take some time downloading and installing Linux.</p>
+      </li>
+      <li>
+        <h2>Start a Linux terminal</h2>
+        <p>Press the Search/Launcher key, type "Terminal", and launch the Terminal app.</p>
+      </li>
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>To install Flatpak, run the following in the terminal:</p>
+        <pre><code>
+          <span class="unselectable">#</span> sudo apt install flatpak
```

Comment:
- Also need to install`xdg-desktop-portal-gtk`
- Need to install from `stretch-backports`. (I don't use Debian but the syntax `flatpak/stretch-backports` might work?)

Reply:
@ahayzen Maybe you know enough about `apt` to chime in?

> and we need `sudo apt install xdg-desktop-portal-gtk` too? because it doesn't get automagically pulled in as a dependency?

It looks like it is a Recommended dep, which AFAIK does get installed by default in Debian so it may actually be fine.

Reply:
On most apt distros i've seen then configure recommends to be auto installed and suggests to not be. I believe the same would happen with backports, so `sudo apt install -t stretch-backports install flatpak` is what I'd expect. If i get a moment in a debian VM i'll give it a try.